modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
Yeah, and that's pretty funky. Do we need access to the TMs `listen` in user space? The `listenRoutes` wouldn't be a bad idea to have access to both on `router`.